modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -84,366 +84,11 @@
 </template>
 
 <script>
-import AposModifiedMixin from 'Modules/@apostrophecms/ui/mixins/AposModifiedMixin';
-import AposEditorMixin from 'Modules/@apostrophecms/modal/mixins/AposEditorMixin';
-import cuid from 'cuid';
-import { klona } from 'klona';
-import { get } from 'lodash';
-import { detectDocChange } from 'Modules/@apostrophecms/schema/lib/detectChange';
+import aposArrayEditorLogic from '../logic/AposArrayEditor';
```

Comment:
We have been capitalizing the first letter of this name as we do with imported mixins in general.

Reply:
Updated!